docs/Config.md

```diff
@@ -411,6 +411,11 @@ os:
   # window is closed.
   editAtLineAndWait: ""
 
+  # Whether lazygit suspends until an edit process returns
+  # Pointer to bool so that we can distinguish unset (nil) from false.
+  # We're naming this `editInTerminal` for backwards compatibility
+  editInTerminal: false
```

Comment:
I agree that it's good that this now shows up in the file. However, only the first line of the comment is for users, the other two are for developers and shouldn't be here.

I pushed a commit that lets us annotate such lines with a tag and filter them out: 1f3050cf9e5.